tests/Core/Tokenizers/Tokenizer/RecurseScopeMapSwitchTokenScopeTest.inc

```diff
@@ -22,3 +22,12 @@ switch ($value):
         break;
 /* testSwitchAlternativeSyntaxEnd */
 endswitch;
+
+// Test for https://github.com/squizlabs/PHP_CodeSniffer/issues/543
+/* testSwitchClosureWithinCondition */
+switch((function () {
+    return 'bar';
+})()) /* testSwitchAfterConditionMarker */ {
```

Comment:
Nitpick: is this marker name clear enough ? Have you considered a name like `testSwitchClosureWithinConditionScopeOpener` to tie the test marker to the code sample and reduce the risk of duplicate marker names in test case files ?

Reply:
Done